src/config/index.js

```diff
@@ -16,14 +20,35 @@ export const TWENTY_FIVE_PERCENT = '25%';
 export const FIFTY_PERCENT = '50%';
 export const SEVENTY_FIVE_PERCENT = '75%';
 export const ONE_HUNDRED_PERCENT = '100%';
+export const AMBROSUS_TOKEN_URL = 'https://token.ambrosus.io';
 
 export const { ethereum } = window;
+
 const host = window.location.hostname;
 
+export const bounce = cssTransition({
+  enter: 'animate__animated animate__bounceIn',
+  exit: 'animate__animated animate__bounceOut',
+});
 export const network =
-  host.includes('local') || host.includes('dev') || host.includes('test')
```

Comment:
https://github.com/ambrosus/staking-frontend/pull/24/files?authenticity_token=s6qprR6Hi%2BDtXe25cj8H%2FsK80XIerzey6EhLeMAEZWGpZOnCXtB1rJpxIh0oX00Mn%2FU8nRL7wcQKHe8YsP%2BNRg%3D%3D&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.scss&file-filters%5B%5D=.svg&file-filters%5B%5D=dotfile#r755062818

Already commented this, of you use same hack in config, it should be better use env variables https://nodejs.dev/learn/how-to-read-environment-variables-from-nodejs